packages/nuxt/src/components/client-fallback-auto-id.ts

```diff
@@ -19,10 +19,10 @@ export const clientFallbackAutoIdPlugin = createUnplugin((options: LoaderOptions
     name: 'nuxt:client-fallback-auto-id',
     enforce: 'pre',
     transformInclude (id) {
-      if (exclude.some(pattern => pattern.test(id))) {
+      if (exclude.length && exclude.some(pattern => pattern.test(id))) {
```

Comment:
Note to self: worth investigating a manual implementation of `some`, as local node benchmarks showed noticeable improvement with a custom `for of` loop and a break statement, which leads me to believe `Array.prototype.some` is O(n) as opposed to worst-case O(n).

Reply:
Still needs to be checked how many elements need to be in an array for it to be beneficial, but looks like the number is unlimited.